opba-protocols/xs2a-protocol/src/test/java/de/adorsys/opba/protocol/xs2a/entrypoint/authorization/Xs2aGetAuthorizationStateViolationsMapperTest.java

```diff
@@ -0,0 +1,41 @@
+package de.adorsys.opba.protocol.xs2a.entrypoint.authorization;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import de.adorsys.opba.protocol.api.dto.ValidationIssue;
+import de.adorsys.opba.protocol.api.dto.result.body.ValidationError;
+import de.adorsys.opba.protocol.xs2a.config.MapperTestConfig;
+import de.adorsys.opba.protocol.xs2a.util.UtilService;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest(classes = MapperTestConfig.class)
+public class Xs2aGetAuthorizationStateViolationsMapperTest {
+    public static final String PATH_PREFIX = "mapper-test-fixtures/validation_error_set_from_";
+
+    @Autowired
+    private Xs2aGetAuthorizationState.ViolationsMapper mapper;
+    @Autowired
+    private UtilService utilService;
+
+    @Test
+    @SneakyThrows
+    public void xs2aGetAuthorizationStateViolationsMapperTest() {
+        // Given
+        Set<ValidationIssue> mappingInput = utilService.getFromFile(PATH_PREFIX + "validation_issue_input.json", new TypeReference<Set<ValidationIssue>>() {
```

Comment:
Can we use more meaningful name - utilService to fixtureProvider

Reply:
UtilService.java was renamed to the FixtureProvider.java
variable renamed correspondingly